frontend/src/metabase/components/EntityMenuItem/EntityMenuItem.tsx

```diff
@@ -51,8 +53,8 @@ const EntityMenuItem = ({
       hoverColor={hoverColor}
       hoverBgColor={hoverBgColor}
     >
-      {icon && <MenuItemIcon name={icon} size={16} />}
-      <MenuItemTitle>{title}</MenuItemTitle>
+      {icon && <MenuItemIcon name={icon} size={16} aria-hidden />}
```

Comment:
icon will appear in many accessible names, when they for decorations we should exclude them from screenreaders.